pkgs/applications/networking/browsers/elinks/default.nix

```diff
@@ -23,7 +23,7 @@ stdenv.mkDerivation rec {
   };
 
   buildInputs = [
-    ncurses xlibsWrapper bzip2 zlib brotli zstd xz
+    ncurses xorg.libX11 bzip2 zlib brotli zstd xz
```

Comment:
Why not to use libX11 directly?

Reply:
I had no idea it was an option. Switched to `libX11`.